src/main/docs/guide/toc.yml

```diff
@@ -237,6 +237,12 @@ cli:
     comparingVersions: Comparing Versions
   features: Features
   commands: Commands
+reloading:
+  title: Reloading
+  jrebel: JRebel
+  springloaded: Spring-Loaded
+  gradleReload: Reloading with Gradle
+  ideReload: Reloading with an IDE
```

Comment:
I don't think this needs to be a top level section. Please make a subclass of the CLI section

Reply:
Agreed it doesn't need to be a top-level section. I will move to the CLI section, however it really isn't a CLI topic (the CLI just helps set up a couple things for reloading - the docs explain how to do it with/without the CLI)